pkg/client/clientcmd/loader.go

```diff
@@ -33,31 +33,35 @@ import (
 const (
 	RecommendedConfigPathFlag   = "kubeconfig"
 	RecommendedConfigPathEnvVar = "KUBECONFIG"
```

Comment:
Recommended* consts for the file names in local and home dirs (`.kubeconfig` and `/.kube/.kubeconfig`)?